website/docs/guides/advanced-cluster-new-sharding-schema.html.markdown

```diff
@@ -0,0 +1,344 @@
+
+```
+resource "mongodbatlas_advanced_cluster" "test" {
+    project_id   = var.project_id
+    name         = "ReplicaSetTransition"
+    cluster_type = "SHARDED"
+
+    replication_specs {
+        region_configs {
+            electable_specs {
+                instance_size = "M10"
+                node_count    = 3
+            }
+            provider_name = "AZURE"
+            priority      = 7
+            region_name   = "US_EAST"
+        }
+    }
+}
+```
+
+Once the cluster type is adjusted accordingly, we can proceed to add a new shard using the new schema:
+
+```
+resource "mongodbatlas_advanced_cluster" "test" {
+    project_id   = var.project_id
+    name         = "ReplicaSetTransition"
+    cluster_type = "SHARDED"
+
+    replication_specs { # first shard
+        region_configs {
+            electable_specs {
+                instance_size = "M10"
+                node_count    = 3
+            }
+            provider_name = "AZURE"
+            priority      = 7
+            region_name   = "US_EAST"
+        }
+    }
+
+    replication_specs { # second shard
+        region_configs {
+            electable_specs {
+                instance_size = "M10"
+                node_count    = 3
+            }
+            provider_name = "AZURE"
+            priority      = 7
+            region_name   = "US_EAST"
+        }
+    }
+}
+```
+
+<a id="leveraging-iss"></a>
+## Leveraging Independent Shard Scaling 
+
+Prerequisite: The new sharding schema must be used, meaning the advanced_cluster configuration is not using `num_shards` and therefore each shard is represented with a unique `replication_specs` element. Please refer to documentation above to transition into this schema if not already.
+
+Considering the following configuration of a SHARDED cluster that has a symmetric configuration for its 2 shards:
+
+```
+resource "mongodbatlas_advanced_cluster" "test" {
+  project_id   = var.project_id
+  name         = "SymmetricShardedCluster"
+  cluster_type = "SHARDED"
+
+  replication_specs { # first shard
+    region_configs {
+      electable_specs {
+        instance_size = "M30"
+        node_count    = 3
+      }
+      provider_name = "AWS"
+      priority      = 7
+      region_name   = "EU_WEST_1"
+    }
+  }
+
+  replication_specs { # second shard
+    region_configs {
+      electable_specs {
+        instance_size = "M30"
+        node_count    = 3
+      }
+      provider_name = "AWS"
+      priority      = 7
+      region_name   = "EU_WEST_1"
+    }
+  }
+}
+```
+
+It is now possible to define a different `instance_size`, and `disk_iops` in the case of AWS, for each individual shard. In the following update we will define an upgraded instance size of M40 only for the first shard. One criteria for defining which shard to scaling independently is by looking into the metrics dashboard in the UI (e.g. https://cloud.mongodb.com/v2/<PROJECT-ID>#/clusters/detail/SymmetricShardedCluster)
```

Comment:
Suggested rephrasing: With each shard's `replication_specs` defined independently, we can now define distinct `instance_size`, and `disk_iops` values for each shard in the cluster. In the following example, we define an upgraded instance size of `M40` only for the first shard in the cluster. 

Consider reviewing the Metrics Dashboard in the MongoDB Atlas UI (e.g. https://cloud.mongodb.com/v2/<PROJECT-ID>#/clusters/detail/SymmetricShardedCluster) for insight into how each shard within your cluster is currently performing, which will inform any shard-specific resource allocation changes you might require.

Reply:
thanks for the suggestion, makes it more clear.